components/RadioPlayer.vue

```diff
@@ -7,6 +7,14 @@
 
       <div class="now-playing-details">
         <div class="radio-controls">
+          <div v-if="showAlbumArt && np.now_playing.song.art" class="now-playing-art">
```

Comment:
is `showAlbumArt` settable somewhere? Could we maybe create a config for the player with list of such options? 

- show album art
- display elapsed time
- display remaining time
- display chatbox
- display `ON AIR` banner

etc. 

That could be really useful for all the different types of things we will be using the player for later (eg. live shows, repeats, shared broadcasts, one-off events etc.)

Reply:
showAlbumArt is the prop we send in (above you had a comment on that line also) – you are right, it is obsolete if we will always use image.
np.now_playing.song.art was done by gammaw with all the if/else cases of availability of image, default azuracast image etc. with fallbacks to show default
nice ideas with other displays, it is also dependable if we have one player for stream/arcsi if we show the chats there... topic for Saturday

Reply:
Good studiment, thank you! Yeah I vaguely remember the now playing logic this will be lot cleaner.